src/Summoner/License.hs

```diff
@@ -80,9 +80,12 @@ customizeLicense l t nm year
             afterN           = T.tail $ T.dropWhile (/= ']') withN
         in  beforeY <> year <> beforeN <> nm <> afterN
 
-fetchLicense :: LicenseName -> IO (Maybe License)
+fetchLicense :: LicenseName -> IO Text
```

Comment:
What is the point of returning `Text`? We shouldn't sacrifice type safety here without any reasons.

Reply:
```
let licenseText = case decodeStrict $ pack licenseJson of
            Just t  -> unLicense t
            Nothing -> error "Broken predefined license list"
    pure licenseText
```
Do you mean to return not `licenseText :: Text` but `t :: License`?